ansible/roles/postgresql11/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
```

Comment:
So the key here is that the role itself will specify the dependencies for other roles, correct?
I think this a reasonable approach, though we should probably document this architecture/pattern somewhere as it might not be immediately obvious this is how roles should be written from now on.